### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by ninadewaal** [rehive/rehive-static-site#69]
![home_feedback](https://user-images.githubusercontent.com/35197761/46730495-97d90180-cc88-11e8-92fa-8749905f9687.png)

**Comment by michailbrynard** [comment]
Somehow it seems that the "Designed for developers"  section is repeated twice on the mobile view.

Also there spacing seems to be too little spacing:

<img width="399" alt="screen shot 2018-10-10 at 1 40 43 pm" src="https://user-images.githubusercontent.com/7025263/46734031-58171780-cc92-11e8-8490-410526cc27a2.png">

Also, we need to look at spacing consistency in general on mobile views.

**Comment by Stephan-e** [comment]
I removed the second appearance of the developers section, I have added some extra padding

**Comment by michailbrynard** [comment]
Not sure if I just missed it before or if it updated as a side effect of improving the standard width page, but the spacing on the mobile view of the homepage seems to be all over the place, compared to say the Platform page which seems to make better use of available space and keep a (generally more) consistent left alignment.

I haven't drawn any red lines, but two examples of what I mean

<img width="350" alt="screen shot 2018-10-10 at 7 04 42 pm" src="https://user-images.githubusercontent.com/7025263/46753290-b4dcf700-ccbf-11e8-93e8-6cb85e11dd44.png">

and

<img width="370" alt="screen shot 2018-10-10 at 7 04 20 pm" src="https://user-images.githubusercontent.com/7025263/46753303-c3c3a980-ccbf-11e8-888e-1218ebb11417.png">

Use of page width also seems to be narrower than on /platform.

**Comment by Stephan-e** [comment]
yeah we did lose it due to the new banner updates that were requested yesterday. 
I will make sure the final version has proper alignment. 

I will look into the landing page for mobile today.
